.goreleaser.yaml

```diff
@@ -0,0 +1,36 @@
+project_name: grid-intensity
```

Comment:
This changes the binary name.

Most of the rest is defaults from `goreleaser init`.